apps/api/src/domain/models/audio-item/commands/create-audio-item/create-audio-item.command-handler.ts

```diff
@@ -41,14 +43,19 @@ export class CreateAudioItemCommandHandler extends BaseCreateCommandHandler<Audi
         mediaItemId,
         lengthMilliseconds,
     }: CreateAudioItem) {
-        return new AudioItem({
+        const createDto = {
             type: AggregateType.audioItem,
             id,
             name,
             mediaItemId,
             lengthMilliseconds,
             published: false,
-        }) as unknown as AudioItem;
+        };
+
+        return buildInstanceFactory(AudioItem as unknown as DomainModelCtor<AudioItem>)(createDto);
+
+        // WARNING The following introduces circular dependencies.
```

Comment:
Remove comment?

Reply:
I'm going to leave it for now until I deal with it properly.